apps/docs/package.json

```diff
@@ -3,19 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "rspress build && pnpm run build-ladle",
-    "build-ladle": "pnpm --filter=@accelint/design-system exec ladle build -o ../../apps/docs/dist/ladle --base ladle",
+    "build:deps": "pnpm run build:ladle && pnpm run build:typedoc",
+    "build:ladle": "pnpm --filter=@accelint/design-system exec ladle build -o ../../apps/docs/src/public/ladle --base ladle",
+    "build:typedoc": "typedoc",
+    "build": "pnpm run build:deps && rspress build",
     "check": "biome check --write",
-    "dev": "rspress dev",
+    "dev": "pnpm run build:deps && rspress dev",
     "format": "biome format --write",
     "preview": "rspress preview"
   },
   "dependencies": {
+    "@accelint/design-system": "workspace:*",
```

Comment:
because the docs build step depends on the design system being built (for ladle), we need to specify it as a dep here. this way, turbo knows that we can only build the docs after the design system has been built. without this, we see build errors like [this](https://github.com/gohypergiant/standard-toolkit/actions/runs/12599681591/job/35117117576?pr=146)